esrally/chart_generator.py

```diff
@@ -1373,12 +1373,13 @@ def load_track(self, cfg, name=None, params=None):
             cfg.add(config.Scope.applicationOverride, "track", "repository.name", self.repository)
             cfg.add(config.Scope.applicationOverride, "track", "track.name", name)
         # another hack to ensure any track-params in the race config are used by Rally's track loader
+        cfg.add(config.Scope.application, "track", "exclude.tasks", excluded_tasks)
```

Comment:
Can we move this down one line as the comment above does not apply here but rather to the line with track parameters?